setup.cfg

```
@@ -71,6 +71,7 @@ install_requires =
   # ansible-lint does not list ansible as direct dependency in order to
   # allow user to choose his target version. Be sure that you mention one
   # of the extras or install a specific version of ansible yourself.
+  ansible-compat>=0.3.0
```

I really think we should stick to a stable version, and not an initial development version which can easily have breaking changes.
```suggestion
  ansible-compat>=1.0.0,<2.0.0
```

---

For 0.x it's usually capped with the next minor:
```suggestion
  ansible-compat >=0.3.0, <0.4.0
```
No need to force a major release just yet — we have a perfect opportunity to iterate on the API and locking it up at this point is a bad idea.
Having the same people in charge of the both projects makes it rather safe. OTOH it may be reasonable to have some downstream tests integrated into the compat's CI to be on the safe side.
Also, zerover does not mean development. Dev versions have suffices like `.dev0`. 0ver just means that the project is not yet fully settled on the API which is not necessarily a bad thing.